genie-common-internal/src/main/java/com/netflix/genie/common/internal/aws/s3/S3TransferManagerFactory.java

```diff
@@ -115,6 +114,8 @@ private S3AsyncClient buildS3AsyncClient(final S3ClientFactory.S3ClientKey s3Cli
         return S3AsyncClient.builder()
             .region(s3ClientKey.getRegion())
             .credentialsProvider(credentialsProvider)
+            .multipartEnabled(true)
```

Comment:
does this `S3AsyncClient` also need to support multi-part file upload:
https://github.com/Netflix/genie/blob/e596781a14cacaeba79893b42e8753609cd3840b/genie-common-internal/src/main/java/com/netflix/genie/common/internal/aws/s3/S3ClientFactory.java#L324
?